com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/handler/AttachRequestHandler.java

```diff
@@ -51,6 +51,7 @@
         context.setAttached(true);
         context.setSourcePaths(attachArguments.sourcePaths);
         context.setDebuggeeEncoding(StandardCharsets.UTF_8); // Use UTF-8 as debuggee's default encoding format.
+        context.setDebugFilters(attachArguments.debugFilters);
```

Comment:
A thought about AttachRequesthandler & LanuchRequestHandler. There are lots of duplicate code from these two classes. We need refactor out a common base in this case.

Reply:
nice. Will add some preProcess and postProcess function to handle the common logic.

Reply:
When implementing launchInTerminal, i have changed the launch logic, would prefer to refactor common base of the attach/launch handler there.
Will send out PR later.